qcelemental/models/multipole.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum
+from typing import Dict, List, Optional
+
+from pydantic import Schema, constr, validator
+
+from .basemodels import ProtoModel
+from .types import Array
+
+
+class Singlepole(ProtoModel):
+    """Model for charges, dipoles, or quadrupoles, etc."""
+
+    angular_momentum: int = Schema(..., description="Angular momentum for this singlepole.")
+    exponents: List[float] = Schema(..., description="Exponents for this singlepole.")
+    geometry: Array[float] = Schema(..., description="Location of the singlepoles.")
+    coefficients: List[List[float]] = Schema(..., description="Coefficients for each AM component. Psi4 AM ordering convention: https://github.com/evaleev/libint/blob/5458ab2fb2fd51dcd19f1e8122a451f2a0808074/doc/progman/progman.tex#L1006-L1008")
```

Comment:
This is the CCA standard, we should probably avoid Psi4 in schema objects.

Is there such a thing as spherical vs cartesian multipoles?